### ISSUE TITLE
Searching a anime is glitched and doesn’t show anything 

### ISSUE THREAD

**Issue by Bannawip** [croberts22/myanilist-bugs#123]
## How can you reproduce it?

1. Go to search
2. Type in any anime
3.  Not a single anime shows up

## How reproducible is this issue?
 all the time even after deleting the app it still shows nothing after redownloading 

## Device Information

iOS Version: 17.5.1

MyAniList Version: 3.2.8

iOS Device: iPhone 15

Jailbroken:

### Screenshots

![IMG_3020](https://github.com/user-attachments/assets/44543395-a50e-4e2b-85e8-e75ffdf31460)

**Comment by Hybridicalz** [comment]
Same thing’s been happening to me since yesterday…
![IMG_5932](https://github.com/user-attachments/assets/794af0b5-09a2-4ad9-8292-5481d7dd1127)
![IMG_5933](https://github.com/user-attachments/assets/c87040ee-ef5b-476a-9880-7b74dbe381e5)
![IMG_5934](https://github.com/user-attachments/assets/04402d79-faf6-4e7c-ac33-b137db439941)

**Comment by Bannawip** [comment]
> Same thing’s been happening to me since yesterday… ![IMG_5932](https://private-user-images.githubusercontent.com/176920934/353085993-794af0b5-09a2-4ad9-8292-5481d7dd1127.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNzM2OTEsIm5iZiI6MTcyMjI3MzM5MSwicGF0aCI6Ii8xNzY5MjA5MzQvMzUzMDg1OTkzLTc5NGFmMGI1LTA5YTItNGFkOS04MjkyLTU0ODFkN2RkMTEyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOVQxNzE2MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZmM0ZDVhNTExYTM2NDNiOWQzYmNmMGZhZGI2MTRmN2Q3OTkwYWMxMzhlYTEzMjQ2ZGZhYjNhN2Q0ZTBiMGVlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.J1GDGPQLIt09DBgFX4xSR90hwmgn8uDRytnxwfo88Jo) ![IMG_5933](https://private-user-images.githubusercontent.com/176920934/353086002-c87040ee-ef5b-476a-9880-7b74dbe381e5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNzM2OTEsIm5iZiI6MTcyMjI3MzM5MSwicGF0aCI6Ii8xNzY5MjA5MzQvMzUzMDg2MDAyLWM4NzA0MGVlLWVmNWItNDc2YS05ODgwLTdiNzRkYmUzODFlNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOVQxNzE2MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NTRiZDI0MDQ5OTBhMjFmMDBiNDgxNzkwMzhiZGJiNDY5NjQ4ZDI2OTRjZWJlYmJiYmU2ZGM2ZjA0OTVkNTk5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.OiXFE5YOJ-g-wIUkb8xg2sQ-yYXGbMk2AHEa45CasWY) ![IMG_5934](https://private-user-images.githubusercontent.com/176920934/353086011-04402d79-faf6-4e7c-ac33-b137db439941.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNzM2OTEsIm5iZiI6MTcyMjI3MzM5MSwicGF0aCI6Ii8xNzY5MjA5MzQvMzUzMDg2MDExLTA0NDAyZDc5LWZhZjYtNGU3Yy1hYzMzLWIxMzdkYjQzOTk0MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOVQxNzE2MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MzkyYzgxMjM5Y2IwMWNkYzBhYjc4YWMxNGJhNmVhZTc0MzQ2ZmQ2MmY0OGQ4MDc5MGI1YWI1M2RkNTljN2Y5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.vYC_STwDFLXhkmFsStYSp0gnBRbhiszXHLELB9Z1TTM)

**Comment by Bannawip** [comment]
@Hybridicalz yeah it gets annoying especially when looking up for new anime to watch and nothing comes out
